types/musicmatch/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "adjacent-overload-signatures": false,
```

Comment:
Please re-enable all lint rules.

Reply:
Thank you for review! I removed unnecessary files from that branch